modules/models/user.php

```diff
@@ -81,4 +80,25 @@ class User extends Record {
 	public $password_digest;
 	public $created_at;
 	public $updated_at;
+
+    // TODO: Make this method more generic and move it
+    // to the Record class
```

Comment:
Why would this be part of the record class? Not all records have the email field right?

Reply:
@chris7neves oh I meant a general "find_by_xxx" static method that I should implement in the record class